admin-dashboard/src/pages/ShieldValidation/index.tsx

```
@@ -0,0 +1,576 @@
+  Select,
+  MenuItem,
+  TextField,
+  FormControl,
+  InputLabel
+} from '@mui/material';
+import {
+  PlayArrow,
+  Stop,
+  Refresh,
+  CheckCircle,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)